src/background/user-activity.ts

```diff
@@ -45,3 +46,10 @@ export async function handleAlarm(alarm: browser.Alarms.Alarm) {
     }
   }
 }
+
+// TODO: use LocallyEncrypted value for {{ address: LocallyEncrypted } | undefined}
+type State = { address: string; walletModelId: string } | null; // Record< | undefined>;
+class LastUsedAddress extends PersistentStore<State> {}
+
+const STORAGE_KEY = 'last-selected-address';
```

Comment:
'last-used-address'?
Not critical, just interesting, why key is different from the name from the beginning?

Reply:
missed after refactoring